tests/data_types_test.ts

```diff
@@ -4,7 +4,7 @@ import { generateSimpleClientTest } from "./helpers.ts";
 import type {
   Box,
   Circle,
-  Float4,
+  // Float4,
```

Comment:
Should we remove it if not used?

Reply:
Eventually, I'm thinking we might use it if we let users add custom parsers  #239 

The way I envision the API for parsers is to have two main parsing modes:
- 'string' : all values are returned as string, and the user has to take care of parsing (`Float4` would be used in this case)
- 'auto' : deno-postgres parses the data into JS objects (as many as possibly implemented [here](https://github.com/denodrivers/postgres/issues/184), non-implemented parsers will return strings)

Both modes are overridable with some API to pass custom parser **_per_** oid type, giving users complete power over how they want any data type to be parsed by the library.

Reply:
That makes sense, thanks!